### PR TITLE
[story/ECP-494] - Personal Details: What gender does your school's payroll system associate with you?

### PR DESCRIPTION
**Pull request by dbrunnellAD**
- Dynamic title sizing added to pre existing file
- Tests added to ECP claim journey

### CODE REVIEW

File: app/models/early_career_payments/slug_sequence.rb [hunk 1]
```diff
@@ -26,6 +26,7 @@ class SlugSequence
       # personal details phase of claim journey
       "how_we_will_use_information_provided",
       "personal-details",
+      "gender",
```
Comment (WorkSutton) [inline app/models/early_career_payments/slug_sequence.rb:29]: I find our use of `gender` and `payroll gender` confusing. Is this the name of the slug for the URL that our UCD colleagues gave us? Or an existing one for the current process in TSLR / M+P?

Comment (dbrunnellAD) [reply app/models/early_career_payments/slug_sequence.rb:29]: The slug in the M&P journey is just labeled as "gender" - I went with sticking to the consistency of the codebase

Comment (WorkSutton) [reply app/models/early_career_payments/slug_sequence.rb:29]: ok lets be pragmatic and we can alter once we have confirmation from UT / AC feedback if indeed it does need to change. I'm all in favour of consistency

**Review by WorkSutton — APPROVED** [review]
LGTM

**Pull request merged**